docker/centrifuge-chain/Dockerfile

```diff
@@ -27,43 +27,41 @@ FROM --platform=linux/amd64 docker.io/paritytech/ci-linux:production as builder
 # and simply download the binaries here. Some short of hash or integrity check would need to
 # happen to make sure the binary is what we want.
 FROM --platform=linux/amd64 docker.io/library/ubuntu:jammy
+	ARG BUILD_DATE
+	ARG VERSION
 
-	LABEL io.centrifuge.image.authors="protocol@k-f.co" \
```

Comment:
LABELS are set using the GHA pipleline on the metadata step. 
Ideally we don't publish images from our laptops and thus no need to have this here